src/ORM/DataObjectSchema.php

```diff
@@ -394,6 +399,10 @@ public function databaseIndexes($class, $aggregated = true)
      */
     public function classHasTable($class)
     {
+        if (!is_subclass_of($class, DataObject::class)) {
+            return false;
+        }
```

Comment:
Somehow `ViewableData` gets passed in here, I haven’t yet tracked down why but this seems like a reasonable change anyway.

Reply:
I think removing the assertion is ok... It's kind of stupid IMO, we probably should have been catching it

Reply:
It's from dataClassesFor() not properly filtering is_subclass_of() the way you have been doing (correctly).